src/main.c

```diff
@@ -465,8 +469,10 @@ int main(int argc, char **argv)
 
 	fts_close(ftsp);
 
-	if (context_path) {
-		paths[0] = context_path;
+	struct string_list *context_path_node = context_paths;
+
+	while (context_path_node) {
```

Comment:
I would use a for-loop, buts that just personal preference

Reply:
Yeah, I've noticed that.  :)  My preference is the while syntax, but I personally have no problems allowing both versions to coexist in the codebase.